### ISSUE TITLE
Completion threshold

### ISSUE THREAD

**Issue by lebaudantoine** [openfun/warren#68]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**
Make accessible the completion threshold of a video to the frontend. This metric was supported in Postie, and designed in the Figma Wireframe.

The completion threshold stands as a value assigned to each video and configured within Marsha. Presently, we employ this value to ascertain the completion status of views and downloads.

**Describe the solution you'd like**

Our ideal solution involves the implementation of a straightforward API endpoint. This endpoint will require the video's unique identifier (UID) as a request parameter. The primary purpose of this API is to procure contextual information to the video, including its completion threshold. 

**Describe alternatives you've considered**

- Experience index and resource's context retrieval: This involves waiting for the experience index and subsequently retrieving the video's context, which encompasses the completion threshold.

- Integration with existing endpoints: Another consideration is to incorporate this functionality into the existing `views` or `download` endpoints.

**Comment by lebaudantoine** [comment]
@quitterie-lcs is this a constant value? Can it be updated in Marsha? 

If the completion threshold is editable, we should fetch the **latest** `initiliazed` statement, and get its value from it.

**Comment by quitterie-lcs** [comment]
Yes, it is a constant value. It is computed for each video in marsha. You can find [here](https://github.com/openfun/marsha/blob/59044d9e7b3420f71bd1945fe561b11e886297a3/src/frontend/packages/lib_components/src/utils/XAPI/VideoXAPIStatement.ts#L143) its computation. It should not be possible to update it, as it is computed with the video duration

**Comment by lebaudantoine** [comment]
Nice !! What do you think of a new route `/{video_id:path}/completion-threshold` for such logic? @quitterie-lcs

**Comment by jmaupetit** [comment]
What about a more generic `/{video_id:path}/info` providing the video name, its duration, completion threshold, etc.?
